exporter.go

```diff
@@ -263,11 +299,28 @@ func (b *Exporter) Listen(e <-chan Events) {
 				eventStats.WithLabelValues("gauge").Inc()
 
 			case *TimerEvent:
-				summary := b.Summaries.Get(
-					b.suffix(metricName, "timer"),
-					prometheusLabels,
-				)
-				summary.Observe(event.Value())
+				timerType := ""
+				if mapping != nil {
+					timerType = mapping.TimerType
+				}
+				if timerType == "" {
+					timerType = b.mapper.Defaults.TimerType
+				}
+
+				if timerType == "histogram" {
```

Comment:
As far as I can see, the timer type string isn't validated at config load time, but probably should be?

And then I'd change this to a `switch timerType {...}` here with a default case that panics. Could be good to extract the strings into named constants too.